packages/cid-verifier/src/verification.js

```diff
@@ -0,0 +1,143 @@
+/* eslint-env serviceworker, browser */
+/* global Response */
+import pRetry from 'p-retry'
+import { getFromDenyList } from './utils/denylist'
+
+/**
+ * Get the google malware result stored in KV.
+ *
+ * @param {string} key
+ * @param {import('./env').Env} env
+ */
+async function getResults (key, env) {
+  const datastore = env.CID_VERIFIER_RESULTS
+  if (!datastore) {
+    throw new Error('CID_VERIFIER_RESULTS db not ready')
+  }
+
+  return await pRetry(
+    () => datastore.get(key),
```

Comment:
i tried using `.list()` but it wasn't returning metadata.  tried adding metadata in the `.put()` calls but no dice.  have we used `.list()` successfully in other repos?

Reply:
figured it out.  had to update the tests.  working now!